shared/middleware/files_api.rb

```diff
@@ -719,6 +719,17 @@ def files_put_file(encrypted_channel_id, filename, body)
   end
 
   #
+  # GET /v3/files-public/<channel-id>/.metadata/<filename>?version=<version-id>
+  #
+  # Read a metadata file, caching the result for 1 hour.
+  #
+  get %r{/v3/files-public/([^/]+)/.metadata/([^/]+)$} do |encrypted_channel_id, filename|
+    cache_for 1 * 60 * 60 # 1 hour
```

Comment:
I see this is where the 1-hour cache gets set, but where is anything set to the 1-minute cache in this PR? Or is the 1-minute caching case discussed in the description already fully implemented in the linked PR, without any further changes in this one?